src/pages/markets/components/markets/_synthetic-indices.tsx

```diff
@@ -59,6 +62,29 @@ const StockIndices = ({ simple_step_content }: StockIndicesProps) => {
                 }
                 display_title={<Localize translate_text="Synthetics trades available on Deriv" />}
             />
+            <Flex.Box
+                direction="col"
+                container="fluid"
+                justify="center"
+                align="center"
+                pb="10x"
+                md={{ pb: '40x', mb: '20x' }}
```

Comment:
just curious, why we have padding and margin bottom? Is it calculate the space?

Reply:
In Figma the space given from the bottom is bigger than 8rem and **40x** is the maximum value we can give to each one of these props. So I used both margin and padding. Let me know if it's not correct 🙏

Reply:
good to know, we can try abuse this. Good work @konstantinosG-deriv